src/app/modules/backbone/components/sidebar/sidebar.component.html

```diff
@@ -3,6 +3,10 @@
         <img class="" src="/favicon.ico"/>
         <span class="">DAK</span>
     </header>
+    <div class="mobile-close" (click)="onMobileCloseClicked()">
+        <div class="mobile-close-bar-1"></div>
+        <div class="mobile-close-bar-2"></div>
+    </div>
```

Comment:
why not use the burger menu component here?

Reply:
You mean give the burger menu an animation to transition into a close-cross so it can fill both roles?

Wouldn't it be confusing for the user to click the sidebar closed, only for the close button to turn into a burger menu during the drawer closing animation? But maybe I'm envisaging it wrong—could you explain in more detail?

Reply:
Yes you are right. I confused the close icon with the burger icon